plugin.rb

```diff
@@ -405,6 +405,23 @@ class ::ListController
     (SiteSetting.assigns_public || scope.can_assign?) && object.topic.assigned_to&.is_a?(Group)
   end
 
+  # SuggestedTopic serializer
+  add_to_serializer(:suggested_topic, :assigned_to_user, false) do
+    DiscourseAssign::Helpers.build_assigned_to_user(object.assigned_to, object)
+  end
+
+  add_to_serializer(:suggested_topic, :include_assigned_to_user?) do
+    (SiteSetting.assigns_public || scope.can_assign?) && object.assigned_to&.is_a?(User)
```

Comment:
Can `object.assigned_to_id` be `nil`? If so, it'll be more efficient to check for the present of that before fetching the `ActiveRecord` object. Also, can't we determine whether it is a user or not from the `assign_type` instead?